assets/js/project.js

```diff
@@ -0,0 +1,48 @@
+import start from '@asl/peephole';
+import debounce from 'lodash/debounce';
+
+const updateProject = project => {
+  return {
+    type: 'UPDATE_PROJECT',
+    project
+  };
+};
+
+const state = window.INITIAL_STATE;
+
+const postData = debounce(data => {
+  window.fetch(state.static.basename, {
+    method: 'PUT',
+    credentials: 'include',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(data)
+  })
+    .then(response => {
+      console.log('ok', response);
+    })
```

Comment:
Pull the console logs pls.

Reply:
![](https://media.giphy.com/media/l46Ce3W4xviv9dxKg/giphy.gif)